packages/asc-ui/src/components/TopTaskButton/TopTaskButton.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Location, Passport, Document } from '@datapunt/asc-assets'
+import TopTaskButton from './TopTaskButton'
+
+export default {
+  title: 'Atoms/TopTaskButton',
+
+  decorators: [
+    (storyFn: () => React.ReactNode) => (
+      <div
+        style={{
+          display: 'flex',
+          maxWidth: 400,
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          padding: '40px 10px',
+        }}
+      >
+        {storyFn()}
+      </div>
+    ),
+  ],
+}
+
+export const OnWhiteBackgroundWithAnchorTag = () => (
+  <>
+    <a href="https://amsterdam.nl" target="_blank" rel="noopener noreferrer">
```

Comment:
Shouldn't TopTaskButton itself be a Button or a Link as a default? Now we need to add css on the div inside for the hover style, that should be on a button or link imho. Then TopTaskButton should be a Button as a default, where you could pass all button (html)props. @jonkoops what do you think?

Reply:
yes I agree, but what to do with redux links?

Reply:
Perhaps use `forwardedAs` with other props?